_posts/2017-09-08-web.md

```diff
@@ -14,26 +14,25 @@ hierarchy:
 
 ## An Introduction 
 
-Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to provide a successful career in web development.
-
+Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to start developing web sites and web appliactions! 
 
 #### How does the Internet work? 
 
-The Internet has become the backbone of our modern-day world, yet few individuals take the time to understand what it is, and how it works. Since it has become such a large part of our lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
+Since the Internet has become a backbone of our modern-day lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
 
-This section will explain the underlying infrastructure and technologies that make the Internet work; it won't provide a very detailed explanation, and resources will be provided at the end for those who wish to learn more. Any comments, questions, suggestions, etc are welcome and may be directed to the author(s) at [team@hackmit.org](mailto:team@hackmit.org).
+This section will explain the underlying infrastructure and technologies that make the Internet work; it won't provide a very detailed explanation, and resources will be provided at the end for those who wish to learn more. Any comments, questions, suggestions, etc are welcome! Check out the [contributing](https://github.com/starthacking/starthacking.github.io/blob/master/CONTRIBUTING.md) section of StartHacking.io! 
```

Comment:
typo: StartHacking.io --> StartHacking.org. You can also just say "Check out the contributing section".

also, could you make the link be https://starthacking.org/contributing.html instead of the GitHub version?